server/server.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"server/api"
+	"time"
+
+	"github.com/gorilla/mux"
+)
+
+func main() {
+	r := mux.NewRouter()
+	r.Methods(http.MethodPost).Path("/api/order").HandlerFunc(api.CreateOrder)
+	srv := &http.Server{
+		Handler: r,
+		Addr:    "0.0.0.0:8000",
+		// Good practice: enforce timeouts for servers you create!
+		WriteTimeout: 15 * time.Second,
+		ReadTimeout:  15 * time.Second,
+	}
```

Comment:
Set IdleTimeout: 60 * time.Second as well?

Reply:
If IdleTimeout is not set, ReadTimeout is used. Let me add a comment for this.    

    // IdleTimeout is the maximum amount of time to wait for the
	// next request when keep-alives are enabled. If IdleTimeout
	// is zero, the value of ReadTimeout is used. If both are
	// zero, there is no timeout.
	IdleTimeout [time](https://pkg.go.dev/time).[Duration](https://pkg.go.dev/time#Duration)